src/components/shared/Help.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const Wrapper = styled.div`
+  position: absolute;
+  top: 200px;
+  left: 50px;
+`
+
+const Button = styled.div`
```

Comment:
could this also be moved into `shared/Button.js` as maybe icon button

Reply:
done